### PR TITLE
Head: Move to SLES15SP4

### PR DESCRIPTION
**Pull request by nodeg**
- Switch the minions to SLES15 SP3 to be able to migrate them to SLES15 SP4.

See https://github.com/SUSE/spacewalk/issues/17410

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-Head-NUE.tf [hunk 1]
```diff
@@ -124,20 +124,22 @@ module "cucumber_testsuite" {
...
     }
     server = {
+      image = "sles15sp4o"
       provider_settings = {
         mac = "aa:b2:93:01:00:b1"
         memory = 10240
       }
     }
     proxy = {
+      image = "sles15sp4o"
```
Comment (srbarrios) [inline terracumber_config/tf_files/SUSEManager-Head-NUE.tf:134]: In theory, that's the default value, we don't need to specify it.

Comment (srbarrios) [reply terracumber_config/tf_files/SUSEManager-Head-NUE.tf:134]: https://github.com/uyuni-project/sumaform/blob/master/modules/proxy/main.tf#L12

File: terracumber_config/tf_files/Uyuni-Master-NUE.tf [hunk 2]
```diff
@@ -128,21 +128,23 @@ module "cucumber_testsuite" {
       }
     }
     server = {
+      image = "sles15sp4o"
```
Comment (srbarrios) [inline terracumber_config/tf_files/Uyuni-Master-NUE.tf:131]: Be caution, Uyuni use opensuse153o for server and proxy.
It did not move yet to opensuse154o

File: terracumber_config/tf_files/SUSEManager-Head-NUE.tf (hunk 1 above)
Comment (nodeg) [reply terracumber_config/tf_files/SUSEManager-Head-NUE.tf:134]: I thought `sles15sp3o` is the default?

Comment (nodeg) [reply terracumber_config/tf_files/SUSEManager-Head-NUE.tf]: Fixed.

File: terracumber_config/tf_files/Uyuni-Master-NUE.tf (hunk 2 above)
Comment (nodeg) [reply terracumber_config/tf_files/Uyuni-Master-NUE.tf]: Fixed.

Comment (nodeg) [reply terracumber_config/tf_files/Uyuni-Master-NUE.tf]: I am wondering here though if we then should update the minions to openSUSE Leap 15.3 at all if we do not plan to use 15.4, yet. If we test the update from 15.2 to 15.3 would't it make sense to migrate this to 15.3 -> 15.4?

Comment (srbarrios) [reply terracumber_config/tf_files/Uyuni-Master-NUE.tf]: To be honest, I'm not sure what we do for Uyuni, we will need to review the code and see if we make any differentiation between Uyuni and SUMA on this feature.

**Comment by nodeg** [comment]
@srbarrios Can we collect here what SLES15 versions our hosts should have?

server: SP4
proxy: SP4
suse-client:
suse-minion: SP3 (since we want to migrate to SP4)
suse-sshminion:
buildhost:
pxeboot-minion:
kvm-host:
xen-host:

**Comment by nodeg** [comment]
Closed in favor of https://github.com/SUSE/susemanager-ci/pull/543 We do not need 2 PRs here.

**Pull request closed without merging**